examples/c++-api/count-stuff.cc

```diff
@@ -0,0 +1,38 @@
+// A demonstration of using khmer to count k-mers; in this case, the request
+// was for exact counting.
+
+#include <vector>
+#include <cmath>
+#include "khmer.hh"
+#include "counting.hh"
+
+using namespace khmer;
+
+int main()
+{
+    unsigned int ksize = 11;
+
+    // for exact counting, you need to create one table that is
+    // >= 4**k.  This will be that size in bytes, note, so you
+    // will need the appropriate amount of memory.
+
+    std::vector<HashIntoType> tablesize;
+    tablesize.push_back(pow(4, ksize));
+    
+    CountingHash ktable(ksize, tablesize);
+
+    ktable.consume_string("ATGGCGATGGCAAGTAGGACCCAGATGGACCAAAG");
+
+    std::cout << "count for: " << "ATGGCGATGGC" << " is " <<
+        ktable.get_count("ATGGCGATGGC") << "\n";
+
+    ktable.count("ATGGCGATGGC");
```

Comment:
Uh, I erroneously assumed `count` was similar in behavior to `get_count`. Seeing #1380 cleared this up for me, that `count` is an alias for `add`.